tests/settings.py

```diff
@@ -12,14 +12,16 @@ def get_ch_version(test_file):
 
 # kubectl_cmd="minikube kubectl --"
 kubectl_cmd = "kubectl"
-test_namespace = "test"
+test_namespace = os.getenv('TEST_NAMESPACE') if 'TEST_NAMESPACE' in os.environ else "test"
```

Comment:
it needs for future, I going to implements parallel test runner each `test_*.py` will run in separate namespace